src/theme/BlogListPage/index.tsx

```diff
@@ -56,6 +56,7 @@ const prioritizedTags: ChipProps["items"] = [
   "engineering",
   "prometheus",
   { label: "k8s", tag: "kubernetes" },
+  "pandas",
```

Comment:
Following the fix in https://github.com/questdb/questdb.io/pull/1791,
Maybe this should be:
```suggestion
  "PANDAS",
```